client/basePaths.ts

```diff
@@ -19,16 +19,17 @@ export const BasePaths = {
     root: (
       countryIso: CountryIso | defaults.countryIso | string = defaults.countryIso,
       assessmentType: AssessmentName | defaults.assessmentType = defaults.assessmentType
-    ) => _generate(countryIso, assessmentType),
+    ) => _generate('assessment', countryIso, assessmentType),
```

Comment:
note:
path fixed as discussed (otherwise it would match any url format `addr.com/a/b`)